src/composed/key.rs

```diff
@@ -555,4 +565,39 @@ mod tests {
 
         assert_eq!(ua.signatures, vec![sig3]);
     }
+
+    #[test]
+    fn test_fingerprint_rsa() {
+        // V4
```

Comment:
The testcases in `test/opengpg-interop/testcases/keys` have json files that contain the right fingerprints.  I plan to parse and test them automatically, but for now you can just hardcode these from there. A single key for each is good enough for now

Reply:
Sure thing, writing them now :)